src/views/resources/form/products.html

```diff
@@ -608,6 +607,17 @@ <h4 class="card-title i18n">
                 </div>
               </div>
             </div>
+            <a class="i18n" href="javascript:;" data-id="ad-relevance-link">
+              <span data-lang="en_us">Config search relevance</span>
+              <span data-lang="pt_br">Configurar relevância na busca</span>
+            </a>
+            <div class="form-group d-none" data-id="ad-relevance">
```

Comment:
```suggestion
            <div class="form-group" style="display: none" data-id="ad-relevance">
```

Pra funcionar com animação via jQuery...